docs/system-admin-guide/authentication/openid-providers/README.md

```diff
@@ -24,7 +24,88 @@ You can configure the following options.
 4. Enter the **Secret**.
 5. Press the blue **create** button.
 
-![Sys-admin-authentication-add-openid-provider](Sys-admin-authentication-add-openid-provider.png)
+
+
+
+
+## Google Workspace
+
+
+
+### Step 1: Create the OAuth consent screen
+
+1. Navigate to your GCP console.  (https://console.cloud.google.com/)
+2. Go to **APIs & Services** > OAuth consent screen.
+
+![image-20220215095201951](/home/ant0n/.config/Typora/typora-user-images/image-20220215095201951.png)
```

Comment:
The paths to the images are not correct, they need to be relative to the README.md file, for example `./images/image-20220215095201951` if you put them in a subfolder `images`

You can configure Typora to move the images relative to the file

![image](https://user-images.githubusercontent.com/459462/154044667-af5353a8-11b8-471e-8672-5d31c8fc0698.png)